torchvision/datasets/widerface.py

```diff
@@ -14,6 +14,8 @@ class WIDERFace(VisionDataset):
     Args:
         root (string): Root directory where images and annotations are downloaded to.
             Expects the following folder structure if download=False:
+            .. code::
```

Comment:
This should fix `.. code::` from appearing in the docs. 

```suggestion

            .. code::
```

Reply:
Great. I will apply the same for Kinetics dataset too.